jekyll/_cci2/authentication.adoc

```diff
@@ -0,0 +1,52 @@
+= Authentication
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+This document describes how to enable, configure, and test CircleCI to authenticate users with OpenLDAP or Active Directory credentials.
+
+NOTE: LDAP is not supported with existing installations, only clean installations may use LDAP.
+
+toc::[]
+
+== Prerequisites
+
+* Install and configure your LDAP server and Active Directory.
+* GitHub Enterprise must be configured and is the source of organizations and projects to which users have access.
+* Install a new instance of CircleCI 2.0 with no existing users.
+* Contact https://support.circleci.com[CircleCI support] and file a feature request for CircleCI Server.
```

Comment:
This prerequisite was a source of some strife in the past, so maybe we should check again if a support ticket is required/expected. I think we want folks to get help with setting this up, but I believe we sell it as a thing anyone can use.

Reply:
Ah right I see, I will try to find out what the recommended process is.